src/main/resources/application.properties

```diff
@@ -1,5 +1 @@
-quarkus.datasource.db-kind=postgresql
-quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/postgres
-quarkus.datasource.username=docker
-quarkus.datasource.password=docker
-quarkus.hibernate-orm.database.generation=update
```

Comment:
@lntrTin0212

Reply:
I removed this properties because its not in my scope, currently i'm using dev service from quarkus to test entity and flyway, you can add it later when implementing feature related to config and connect database